### ISSUE TITLE
How to train

### ISSUE THREAD

**Issue by li-an-sheng** [lukemelas/EfficientNet-PyTorch#335]
Hello, I want to ask how I can train my own data set now. I don't see the training code

**Comment by ayicherry** [comment]
See examples/imagenet/main.py

**Comment by Riddhasree** [comment]
Do we need to always normalize using the Imagenet mean/std values (normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])) when we are using an Imagenet pretrained Efficientnet model from here for training on our different dataset? I have read that the pretrained models available in Pytorch needs this normalization as a preprocessing step (https://discuss.pytorch.org/t/how-to-preprocess-input-for-pre-trained-networks/683).
